### DISCUSSION TITLE
Examples

### DISCUSSION THREAD

**Discussion by tripleo1** [pen-lang/pen#386] [General]
Will you provide examples?

Like Rosetta Code, Are we fast yet or Todo MVC?

Or anything. I want to get a better sense of what the language looks like.

**Comment by raviqqe** [comment]
We already have a bunch of examples in the documentation. But do you think they are too simplistic?

- https://pen-lang.org/examples

**Comment by tripleo1** [comment]
Yes, I seen those examples, but I was looking for something more of a full program, one that would demonstrate the syntax and capabilities of the language, which I find interesting from first glance.

One, sufficiently complex, would be enough.  Or you could implement some from the above.

**Comment by raviqqe** [comment]
I just made [an issue](https://github.com/pen-lang/pen/issues/401) for it! Thank you for your feedback!

**Comment by tripleo1** [comment]
Thank you.

**Comment by raviqqe** [comment]
I just added some small but full application examples in the code base [here](https://github.com/pen-lang/pen/tree/main/examples). More is coming up when the standard packages are extended with networking functionalities!
